widgyts/dataset_viewer.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+import json
+import pythreejs
+import traitlets
+import ipywidgets
+from IPython.display import display, JSON
+import matplotlib.cm as mcm
```

Comment:
This is imported also by the colormap widget. Maybe we could abstract some of that away to be used by both the colormap widget and this in a base utility function (or that's overengineering!) ?